tools/build-docs

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd docs
+rm -rf _build
+make html
+cd ..
+s
```

Comment:
When writing shell scripts like this, you should never assume anything about the current working directory that the user is running it from, because it'll fail for those users.  See `tools/test-all` for an example: It does `cd "$(dirname "$0")"/..` at the top to address this.

Also, what's the `s`?

(As a follow-up project, it may be worth doing a quick audit from the shell scripts under `tools/` for this issue; `tools/test-migrations` at least has it as well)